docs/user/section-2-podman.md

```diff
@@ -0,0 +1,306 @@
+# Section 2 - Installing Perses in a Container
+
+This sections continues with installing using a container image with Podman open source container tooling, configure 
+examples, and run the container on your local machine.
+
+## Installing Podman tooling
+
+Installing Perses using a container image is going to be demonstrated here using the 
+[open source project Podman](https://podman.io/). It's assumed you have already installed the Podman commandline tooling 
+previously.
+
+If you want to use other container tooling, such as Docker, it's left to the student to determine the commands to 
+substitute as needed.
+
+## Starting Podman machine
+You need to make sure that the Podman virtual machine has been started, so, assuming you have initialized a podman 
+machine already:
+
+```shell
+$ podman machine start
+```
+
+## Running container image
+It's pretty straight forward to running Perses in a container, just start the Perses container image as follows:
+
+```shell
+$ podman run --name perses -d -p 127.0.0.1:8080:8080 persesdev/perses:v{VERSION-NUMBER}
+```
+
+The details in this command are that we give the container a referencable name (--name perses), detach the container 
+from the command line (-d), map the local machine port 8080 to the container port 8080 (-p 127.0.0.1:8080:8080), and use 
+the image version supported in these instructions (persesdev/perses:v{VERSION-NUMBER}). Note: you can use any local port you have 
+available, but you need to map to the container port 8080.
+
+### Note: What do do with failures?
+
+At anytime in these instructions should you encounter failures, during installation, testing, data population, or build 
+results. Don't worry, this can be rerun anytime after you fix any problems reported. You might have to remove the perses 
+container depending on how far you get before something goes wrong. Just stop, remove, and restart it:
+
+```shell
+$ podman container stop perses
+
+$ podman container rm perses
+
+$ podman run --name perses -d -p 127.0.0.1:8080:8080 persesdev/perses:v{VERSION-NUMBER}
+```
+## Connect a browser (default)
+
+Open the Perses console at http://localhost:8080.
+
+You are presented with an empty home dashboard, in light mode. For fun you can choose optionally to flip the switch in
+the top right corner to enable dark mode.
+
+## But this version is empty?
```

Comment:
I agree with @karsov, these parts shouldn't be in a user documentation.

On the contrary starting from scratch will give the possibility to introduce various notion, such as datasource / global variable, global secrets and of course dashboard.

That's a user documentation, so as a user I'm expecting to find information about how to install it of course, but after how I can use the UI, the CLI, the data provisioning potentially etc. 

Basically we should find information about any features Perses is providing.

And then if there is a third party that proposes additional content around Perses then yeah we can reference it if it makes sense.

Reply:
This is all coming in the following sections, 3 and 4 specifically, after installation. This is just to get something up easily and ready for user to explore. After showing the API, CLI, UI, etc, then user designs a simple dashboard, then adds to that, and finally ends up with a complex dashboard.

Reply:
Removed usage of install project.